src/app/js/public/graph/sagas.js

```diff
@@ -76,15 +83,22 @@ export function* handleLoadChartDataRequest({ payload: { field } }) {
             value: item.docCount,
         }));
         yield put(loadChartDataSuccess({ name, data }));
+        return;
     }
-
-    yield delay(500);
+    yield put(loadChartDataSuccess({ name, data: null }));
 }
 
 export default function*() {
     yield fork(function*() {
-        yield takeEvery(
-            [LOAD_CHART_DATA, TOGGLE_FACET_VALUE, CLEAR_FACET, APPLY_FILTER],
+        yield throttle(
+            500,
+            [
```

Comment:
I can't find in the `redux-saga` doc the part about `throttle` taking an array as second argument. Can you add a link in a JS comment?